content/applications/general/voip/axivox/dial_plan_advanced.rst

```diff
@@ -0,0 +1,289 @@
+==================
+Dial plan advanced
+==================
+
+Companies have a lot of incoming calls every day, but many do not want their teams to answer calls
+24 hours a day, 7 days a week. Using *Axivox* advanced dial plan features, the process can be
+automated and routing can be setup for all scenarios. This way customers are never left waiting or
+frustrated because they can not get in touch with anyone.
+
+Using the advanced elements in dial plans, companies can automate call routing for certain days or
+times, like company holidays. Companies can also allow callers to enter extensions themselves and
+get transferred automatically using a digital receptionist, so an administrative team does not have
+to be available around the clock. There is even the option to route callers depending on where they
+are calling from in the world, to maximize efficiency.
+
+.. seealso::
+   For more information on basic dial plans and adding elements visit :doc:`dial_plan_basics`.
+
+Advanced elements
+=================
+
+In Axivox dial plans there are two advanced elements that can be used.
+
+- :guilabel:`Record`: recording feature is enabled (requires plan change, enabled in Settings).
+- :guilabel:`Caller ID`: replace the caller ID by the called number or free text.
+
+To add one of these elements navigate to :guilabel:`Dial plans` in the left menu of the `Axivox
+management console <https://manage.axivox.com>`_. The :guilabel:`Record` element records calls that
+are routed through this element and requires an additional plan change in Axivox.
+
+To enable recording on Axivox, navigate to :guilabel:`Settings` in the `Axivox management console
+<https://manage.axivox.com>`_. Then go to the :guilabel:`Recording` drop-down menu near the bottom
+of the page. Select :guilabel:`Enabled` to enable recording using the :guilabel:`Record` element in
+a dial plan.
```

Comment:
```suggestion
To enable recording on Axivox, navigate to :guilabel:`Settings` in the `Axivox management console
<https://manage.axivox.com>`_. Then, go to the :guilabel:`Recording` drop-down menu, near the bottom
of the page. From there, select :guilabel:`Enabled` from the drop-down menu to enable recording
using the :guilabel:`Record` element in a dial plan.
```

Reply:
It might be smart to explain why this Recording drop-down menu might not be editable/changeable in the Settings page, as it's currently grayed-out and unable to change.